python_modules/libraries/dagster-celery-k8s/dagster_celery_k8s/config.py

```diff
@@ -31,7 +31,6 @@ def celery_k8s_executor_config():
         "job_namespace": Field(
             StringSource,
             is_required=False,
-            default_value="default",
```

Comment:
whats up with swapping out this default?

Reply:
If we didn't swap out the default, it would always be set, and always override whatever was set on the run launcher